pkg/shortnames/shortnames.go

```diff
@@ -149,9 +149,9 @@ const (
 func (r *Resolved) Description() string {
 	switch r.rationale {
 	case rationaleAlias:
-		return fmt.Sprintf("Resolved short name %q to a recorded short-name alias (origin: %s)", r.userInput, r.originDescription)
+		return fmt.Sprintf("Resolved %q as an alias (%s)", r.userInput, r.originDescription)
 	case rationaleUSR:
-		return fmt.Sprintf("Completed short name %q with unqualified-search registries (origin: %s)", r.userInput, r.originDescription)
+		return fmt.Sprintf("Resolving %q using unqualified-search-registries (%s)", r.userInput, r.originDescription)
```

Comment:
```suggestion
		return fmt.Sprintf("Resolving %q using unqualified-search registries (%s)", r.userInput, r.originDescription)
```
The 2nd dash isn't needed.

Reply:
The option in the config file is literally named `unqualified-search-registries`.

Reply:
For technical reasons though.  The hyphen in English is needed to relate "search".